package-lock.json

```diff
@@ -1,8 +1,2676 @@
 {
   "name": "privacy-test-pages",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
oh no, not this again. If we are switching to lockfile v2 we should add a hard requirement for new npm/node (14+?)